services/src/main/java/org/keycloak/services/resources/admin/AuthenticationManagementResource.java

```diff
@@ -729,6 +746,7 @@ public void recurseExecutions(AuthenticationFlowModel flow, List<AuthenticationE
     @Consumes(MediaType.APPLICATION_JSON)
     @Tag(name = KeycloakOpenAPI.Admin.Tags.AUTHENTICATION_MANAGEMENT)
     @Operation( summary = "Update authentication executions of a Flow")
+    @APIResponse(responseCode = "202", description = "Accepted", content = @Content(schema = @Schema(implementation = AuthenticationFlowRepresentation.class)))
     public Response updateExecutions(@Parameter(description = "Flow alias") @PathParam("flowAlias") String flowAlias, @Parameter(description = "AuthenticationExecutionInfoRepresentation") AuthenticationExecutionInfoRepresentation rep) {
```

Comment:
One comment here. Previously the flow model was returned. The PR changes to return the `AuthenticationFlowRepresentation`, but maybe it makes no sense. We can better return the same `AuthenticationExecutionInfoRepresentation` or something related to the execution and not the flow. I just returned the flow rep to be more similar to previous behavior, but I suppose it's not used by anybody, so we can change it now.

Reply:
I don't have any preference here. Our other "update" methods in REST API usually return status 204 with no content (EG. methods `org.keycloak.services.resources.admin.ClientResource.update` or `org.keycloak.services.resources.admin.UserResource.updateUser` ), so I can imagine we return 204 from this one as well?

But if you prefer to keep as it is in your PR (with flow returned) or return somehow the updated executions, it s fine with me.

Reply:
Let me try it. Maybe some test fails if we change to 204. I'll test the console too.

Reply:
I see no issues so I have changed to use void/204 instead of accept.